.github/workflows/deploy-test-pypi.yml

```
@@ -0,0 +1,18 @@
+name: Publish to TestPyPI
+
+on:
+  push:
+    branches:
+      - 'release*'
```

I think this needs a slash? Or a dash?
```suggestion
      - 'release/*'
```

---

I did `release/*.*.*` for consistency with the tag filter.